php/auth.php

```diff
@@ -0,0 +1,49 @@
+<?php
+$ERRORLOG = getenv('PHP_ERROR_LOG');
```

Comment:
Would `ini_get('error_log');` work the same? I would rather not have to make the web interface wait for an update from Pi-hole Core.